uint64.h1

```diff
@@ -0,0 +1,15 @@
+/* Public domain. */
+
+#ifndef UINT64_H
+#define UINT64_H
+
+/* sysdep: -ulong64 */
```

Comment:
I would just kill this and ```#include <stdint.h>``` and then use uint64_t, see #30.

Reply:
We have merged #30 meanwhile, so pleasekill all of that typedefs and just use uint64_t from stdint.h.

Reply:
Done.

Reply:
not the second part ;)

Reply:
This commit was pushed to be in alignment with PR #30 and the uint32.h header file which typedefs the uint32 datatype to uint32_t. Replacing uint64 entirely in favor of uint64_t can also be done once I walk through all of the tai_*.[ch] files and fix the includes.

Reply:
We have not done this in #30 as it would cause collisions with other patches as many files are touched. Here all users of that type are your new code anyway so there is nothing this can collide with.